docs/en/cloud/manage/openapi.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: Control Plane Open API
+slug: /en/cloud/manage/openapi
+title: Control Plane Open API
+---
+
+ClickHouse Cloud provides an API utilizing OpenAPI that allows you to programmatically manage your account and aspects of your services.
+
+:::important This feature is currently experimental and only available by request. Please reach out to support to enable the `FT_USER_OPENAPI_KEYS` feature:::
+
+# Managing API Keys
+
+:::note
+This document covers the Control Plane API. For database API endpoints, please see [Cloud Endpoints API](/docs/en/cloud/security/cloud-endpoints-api.md)
+:::
+
+1. You can use the **API Keys** tab on the left menu to create and manage your API keys.
+
+  ![ClickHouse Cloud API Keys Tab](@site/docs/en/_snippets/images/openapi1.png)
+
+2. The **API Keys** page will initially display a prompt to create your first API key as shown below. After your first key is created, you can create new keys using the `New API Key` button that appears in the top right corner.
+
+  ![Initial API Screen](@site/docs/en/_snippets/images/openapi2.png) 
+  
+3. To create an API key, specify the key name, permissions for the key, and expiration time, then click `Generate API Key`.
```

Comment:
Should we be more clear about the differences between Admin and Developer here?
I think in general Developer is more like a readonly permission.
We should definitely add `Minimum Required Permission` to our specs - @mfursov